scripts/Tools/cs.status

```diff
@@ -1,80 +1,100 @@
 #!/usr/bin/env python
 """
-List test results based on TestStatus files. Returns True if
-no errors occured (not based on test statuses).
+List test results based on TestStatus files.
+
+Typical usage:
+    ./cs.status /path/to/testroot/*.testid/TestStatus
+
+Returns True if no errors occured (not based on test statuses).
 """
 
 from standard_script_setup import *
 import argparse, sys, os, logging, glob
-from CIME.test_status import *
+from CIME.utils import expect
+from CIME.cs_status import cs_status
```

Comment:
I seem to recall there was a reason not to give files and classes the same name, but don't remember what it was - do you?

Reply:
I can't remember, but I can rename either the module or the function if you think I should.

Reply:
After I made the comment I realized that it's a function and not a class so I think this is okay.